jenkins/e2e-image/e2e-runner.sh

```diff
@@ -378,6 +378,7 @@ if [[ -n "${BOOTSTRAP_MIGRATION:-}" ]]; then
   # pattern, but until then we also need jobs to continue uploading started.json
   # until that time. This environment variable will do that.
   source "$(dirname "${0}")/upload-to-gcs.sh"
+  version=$(find_version)  # required by print_started
```

Comment:
OK, that further explains why the hash doesn't exist, since this commit appears after the image commit.